pages/chat/_slug.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <TLoader v-if="loading" />
   <div v-else class="flex flex-col h-full">
-    <THeader :title="`Chat with ${receiver.username}`" />
+    <THeader :title="$t(`chat.message.receiver, ${receiver.username}`)" />
```

Comment:
That won't work. Here is the right way:

```suggestion
    <THeader :title="$t('chat.message.receiver', { username: receiver.username })" />
```

Reply:
What's the difference between chat, conversation and message?